subsys/zigbee/cli/zigbee_cli.c

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <init.h>
+#include <shell/shell.h>
+#include <shell/shell_uart.h>
+#include <shell/shell_rtt.h>
+
+#include <zb_nrf_platform.h>
+#include "zigbee_cli.h"
+
+/* CLI Agent endpoint, by default set to greatest endpoint number. */
+static zb_uint8_t cli_ep = ZB_MAX_ENDPOINT_NUMBER;
```

Comment:
The idea is to have easy Zigbee CLI easy pluggable for any application, if endpoint to be used by the cli is not specified, from all defined endpoints, endpoint with lowest number is used.
However, user can set endpoint in application and `zb_set_default_cli_endpoint` will not overwrite it. Similarly, device handler will not be overwritten if set by the application already (so far, this might be changed a bit later).